stub/include/neurala/video/VideoSourceStatus.h

```diff
@@ -29,60 +29,46 @@
 namespace neurala
 {
 /**
- * @brief Enum that describes the status of a VideoSource
+ * @brief Type that describes the status of a VideoSource
  */
-enum class VideoSourceStatus
+struct VideoSourceStatus final
```

Comment:
Any error code you think we should add here? I noticed you used the pixel format one and the overflow one not exactly the way it's intended in the C# plugin.